app/kitchensink/kitchensink.js

```diff
@@ -124,4 +124,10 @@ $(document).ready(function () {
   $('#topSecret').on('click', function () {
     hulkIt('top-secret')
   })
+
+  //brand card interaction
```

Comment:
could we provide a way to contract as well?

Reply:
Added.